src/main/java/com/exadel/hotel/type/controller/TypeController.java

```diff
@@ -26,21 +27,23 @@ public ResponseEntity<TypeDto> get(@PathVariable("id") Long id) {
 
     @PostMapping("/add")
     public ResponseEntity<TypeDto> add(@RequestBody TypeDto typeDto) {
-        return ResponseEntity.ok(typeService.add(typeDto));
+        return new ResponseEntity<>(typeService.add(typeDto), HttpStatus.CREATED);
     }
 
     @PutMapping("/update/{id}")
     public ResponseEntity<TypeDto> update(@PathVariable("id") Long id, @RequestBody TypeDto typeDto) {
-        return ResponseEntity.ok(typeService.update(id, typeDto));
+        return new ResponseEntity<>(typeService.update(id, typeDto), HttpStatus.ACCEPTED);
     }
 
     @DeleteMapping("/delete/{id}")
-    public void delete(@PathVariable("id") Long id) {
+    public ResponseEntity<Boolean> delete(@PathVariable("id") Long id) {
```

Comment:
It's a bit strange, you don't send response body.
Let's go to do
 public ResponseEntity<Void> delete(@PathVariable("id") Long id) {
        typeService.delete(id);
        return new ResponseEntity<>(HttpStatus.OK);
    }

Reply:
in this case sonarlint give me advise that I should add ResponseEntity type

Reply:
Or if it is possibe and nothing to worth I will change it